### PR TITLE
release: v0.79.0 CHANGELOG

### PR DESCRIPTION
**Pull request by mboersma**
Add CHANGELOG for upcoming v0.79.0 release

### CODE REVIEW

File: releases/CHANGELOG-v0.79.0.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+<a name="v0.79.0"></a>
+# [v0.79.0] - 2023-08-29
+### Features 🌈
+- add support for Kubernetes v1.24.17 ([#5112](https://github.com/Azure/aks-engine/issues/5112))
```
Comment (CecileRobertMichon) [inline releases/CHANGELOG-v0.79.0.md:5]: we're missing the Azure CNI bump

Comment (CecileRobertMichon) [reply releases/CHANGELOG-v0.79.0.md:5]: https://github.com/Azure/aks-engine/commit/3970026335e1ef85ef2c41b22164074a31d3da77

Comment (mboersma) [reply releases/CHANGELOG-v0.79.0.md:5]: I added it in the "Maintenance" section.

**Review by CecileRobertMichon — APPROVED** [review]

**Review by bridgetkromhout — APPROVED** [review]
LGTM

**Pull request merged**